pkg/coreapi/coreapi.go

```diff
@@ -73,3 +90,71 @@ func (a *CoreAPI) Start(ctx context.Context) error {
 func (a CoreAPI) Stop(ctx context.Context) error {
 	return a.server.Shutdown(ctx)
 }
+
+func (a CoreAPI) EventRuns(w http.ResponseWriter, r *http.Request) {
+	if a.tracker == nil {
+		_ = publicerr.WriteHTTP(w, publicerr.Error{
+			Status:  500,
+			Message: "No tracker",
```

Comment:
nit: not quite sure how actionable this error message is

Reply:
Agreed, it's not — kind of by design?  This should never happen, and if we see it we'll get a report and know something is extremely broken.  It's an internal error for our testing and can't happen to a user.